crates/query-engine/translation/src/translation/mutation/experimental/generate.rs

```diff
@@ -3,26 +3,36 @@
 use super::delete::{generate_delete_by_unique, DeleteMutation};
 use super::insert;
 use super::insert::InsertMutation;
+use super::update::{generate_update_by_unique, UpdateMutation};
 use query_engine_metadata::metadata::database;
 use std::collections::BTreeMap;
 
 #[derive(Debug, Clone)]
 pub enum Mutation {
     DeleteMutation(DeleteMutation),
     InsertMutation(InsertMutation),
+    UpdateMutation(UpdateMutation),
```

Comment:
How does this change interact with versioning?

IIUC, If I have enabled mutations in my ndc configuration (i.e., a non-null `mutationsVersion`?) I now also get update mutations generated automatically.

Should we have a `mutations-version: v3` that includes updates? 
Can a 'local' user access these update mutations?

(FWIW, I think I'm fine if the answer is simply that we consider mutations an unstable, experimental, unreleased, work in progress and users shouldn't expect to be able to try them out anywhere but on cloud.)

Reply:
If you set your mutationsVersion to `veryExperimentalWIP` (or whatever it is called), these update mutations will be generated as part of the ndc-postgres-cli `update` command.

We do not release this feature yet for general consumption. The same is true for delete and insert as well. Once we decide to do it, we will rename the whole `experimental` module to `v2`.